OpenCL_C.txt

```diff
@@ -16289,7 +16289,7 @@ is the infinitely precise result.
 | *_x_ - _y_*   | Correctly rounded             | Correctly rounded
 | *_x_ * _y_*   | Correctly rounded             | Correctly rounded
 | *1.0 / _x_*   | Correctly rounded             | \<= 1 ulp
```

Comment:
Discussed in the January 14th teleconference, and we decided to relax reciprocal (for now) as well.

```suggestion
| *1.0 / _x_*   | \<= 1 ulp                     | \<= 1 ulp
```